public/app/features/search/page/components/SearchView.tsx

```diff
@@ -42,7 +42,7 @@ export const SearchView = ({ showManage, folderDTO, hidePseudoFolders, keyboardE
   const [listKey, setListKey] = useState(Date.now());
 
   // Search usage reporting
-  useDebounce(stateManager.onReportSearchUsage, 1000, []);
+  useDebounce(() => stateManager.onReportSearchUsage(), 1000, []);
```

Comment:
since it's named onXX I would expect it to be a instance property with arrow function, so an alternative fix would be to fix that name it an instance function with arrow

Reply:
Hmm, im not sure i follow/understand that reasoning, but I guess i come to the same conclusion. Looking at SearchStateManager, this `onReportSearchUsage` is the only function there that _isnt_ an arrow function.